projects/storefrontlib/cms-components/product/product-list/product-grid-item/product-grid-item.component.html

```diff
@@ -37,9 +37,11 @@
 </ng-template>
 
 <ng-template [cxOutlet]="ProductListOutlets.ITEM_ACTIONS">
+  <!--
```

Comment:
Is fixing this already covered in an another ticket?

Reply:
in ticket #13050